debian/watch

```diff
@@ -0,0 +1,3 @@
+version=4
+opts=filenamemangle=s/.+\/v?(\d\S+)\.tar\.gz/apt-transport-in-toto-$1\.tar\.gz/ \
```

Comment:
I didn't manage to add a working `pgpsigurlmangle` option as described in the corresponding [debian/watch wiki entry](https://wiki.debian.org/debian/watch#GitHub). Detached signatures over the upstream source release (tar.gz) will be made available here:
```
https://github.com/in-toto/apt-transport-in-toto/releases/download/v<version number>/v<version number>.tar.gz.asc
```


FYI I tried debugging with `uscan` dry run, like so:
```bash
uscan --watchfile watch --package apt-transport-in-toto --upstream-version 0.1.0 --no-download --verbose
```

Maybe someone can help here?